aiohttp/cookiejar.py

```diff
@@ -62,9 +62,10 @@ class CookieJar(AbstractCookieJar):
     )
     try:
         calendar.timegm(time.gmtime(MAX_TIME))
-    except OSError:
+    except (OSError, ValueError):
         # Hit the maximum representable time on Windows
         # https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/localtime-localtime32-localtime64
+        # Throws ValueError on PyPy 3.8, OSError elsewhere
```

Comment:
Seems like it also throws the same error on Windows/PyPy-3.9. The latest commit in https://github.com/psf/black/pull/4035 bumps PyPy to 3.9 in black's CI, but there's still a `ValueError` being raised in the Windows/PyPy job: https://github.com/psf/black/actions/runs/6915180334/job/18813836767?pr=4035